Products/ZenModel/migrate/AddHMasterRegionServerGraphConfigs.py

```diff
@@ -0,0 +1,490 @@
+
+            if not filter(lambda x: x.graphID == 'opcounts', gcs):
+                log.info("No opcounts graph found; creating.")
+                commit = True
+                gcs.append(
+                    GraphConfig(
+                        graphID='opcounts',
+                        name='Operation Counts',
+                        footer=False,
+                        returnset='EXACT',
+                        graphType='line',
+                        miny=0,
+                        yAxisLabel='Operations',
+                        description='Slow Operations',
+                        graphRange=GraphRange(
+                            start='10s-ago',
+                            end='0s-ago'
+                        ),
+                        datapoints=[
+                            GraphDatapoint(
+                                aggregator='zimsum',
+                                fill=True,
+                                pointID='totalRequestCount',
+                                legend='The total number of requests received',
+                                metric='totalRequestCount',
+                                metricSource='RegionServer',
+                                name='The total number of requests received',
+                                rate=False,
+                                pointType='line'
+                            ),
+                            GraphDatapoint(
+                                aggregator='zimsum',
+                                fill=True,
+                                pointID='readRequestCount',
+                                legend='The total number of read requests received',
+                                metric='readRequestCount',
+                                metricSource='RegionServer',
+                                name='The total number of read requests received',
+                                rate=False,
+                                pointType='line'
+                            ),
+                            GraphDatapoint(
+                                aggregator='zimsum',
+                                fill=True,
+                                pointID='writeRequestCount',
+                                legend='The total number of write requests received',
+                                metric='writeRequestCount',
+                                metricSource='RegionServer',
+                                name='The total number of write requests received',
+                                rate=False,
+                                pointType='line'
+                            )
+                        ]
+                    )
+                )
+            else:
+                log.info("opcounts graph found; skipping.")
+
+            if not filter(lambda x: x.ID == 'RegionServer', mcs):
+                log.info("No RegionServer internal metrics found; creating.")
+                commit = True
```

Comment:
I believe that `commit = True` should be after append() method, move it to the 481 line
similar in other parts.